### PR TITLE
StorageSrv reset implemented

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

File: app/scripts/services/storage.srv.js [hunk 1]
```diff
@@ -56,6 +56,11 @@ angular.module('dashboard')
                 else {
                     $log.error("StorageSrv: deleteKey invalid parameter");
                 }
+            },
+            reset: function () {
+                for (var key in data) {
+                    delete data[key];
```
Comment (tkenakka) [inline app/scripts/services/storage.srv.js]: Why delete, why not e.g. replace `data` using a new empty object?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete

Comment (evidentti) [inline app/scripts/services/storage.srv.js]: why not? It does same as deleteKey function above.

Comment (tkenakka) [inline app/scripts/services/storage.srv.js]: deleteKey is for deleting one specific property.

Comment (tkenakka) [inline app/scripts/services/storage.srv.js]: deleteKey is for deleting one specific property.

Comment (tkenakka) [inline app/scripts/services/storage.srv.js]: deleteKey is for deleting one specific property.

Comment (tkenakka) [inline app/scripts/services/storage.srv.js]: No biggie, just that deleteKey is for deleting one specific property. In principle replacing using an empty object sounds more simple than iterating it through.

**Comment by tkenakka** [comment]
LGTM

**Pull request merged**